game-core/src/main/java/games/strategy/triplea/player/AbstractBasePlayer.java

```diff
@@ -54,20 +58,17 @@ public void start(final String stepName) {
       // it.
       String bridgeStep = getPlayerBridge().getStepName();
       int i = 0;
-      boolean shownErrorMessage = false;
       while (!stepName.equals(bridgeStep)) {
         Interruptibles.sleep(100);
         i++;
-        if (i > 30 && !shownErrorMessage) {
-          System.out.println("Start step: " + stepName + " does not match player bridge step: " + bridgeStep
+        if (i == 30) {
```

Comment:
Why was this if check changed?

Reply:
Simplification, notice that `shownErrorMessage` drops out. To be fully accurate to previous this should have been `31` now that I look at it. The previous was also slightly 'wrong' to show message at 31, 0 is inclusive so the message would have been shown at the start of the 33rd second of waiting :man_shrugging: 

The point here though was that it's more direct to say when i == X, instead of the extra flag that is set to false whenever i > X